geoextent/lib/extent.py

```diff
@@ -99,38 +102,44 @@ def run(self):
             elif self.task == "tbox":
                 try:
                     if tbox:
-                        metadata["tbox"] = usedModule.getTemporalExtent(filePath)
+                        if usedModule.fileType == 'text/csv':
+                            extract_tbox = usedModule.getTemporalExtent(filePath, num_sample)
+                        else:
+                            if num_sample is not None:
+                                logger.warning("num_sample parameter NOT used, only required for csv")
```

Comment:
Is this really a "warning"?

Reply:
Please rephrase to "num_sample parameter is ignored, only applies to CSV"

Reply:
Raise warning if API user adds a `num_sample` parameter for a file that does not require it. The  `num_sample` parameter is only required for 'csv'. For other files this value is ignored.